File: 프로그래머스/lv2/12949. 행렬의 곱셈/행렬의 곱셈.py
def solution(arr1, arr2):
    N = len(arr1)    # 앞에거 세로길이
    M = len(arr2[0]) # 뒤에거 가로길이
    K = len(arr1[0]) # 앞에거 가로길이
    
    answer = [[0 for _ in range(M)] for _ in range(N)]
    
    for n in range(N): 
        for m in range(M): 
            for k in range(K): 
                answer[n][m] += arr1[n][k] * arr2[k][m]
    return answer

File: 프로그래머스/2/12949. 행렬의 곱셈/행렬의 곱셈.js
function solution(arr1, arr2) {
    const n = arr1.length;
    const m = arr2[0].length;
    const k = arr1[0].length;
    
    const answer = Array.from(Array(n), () => Array(m).fill(0));
    for (let i = 0; i < n; i++) {
        for (let j = 0; j < m; j++) {
            for (let t = 0; t < k; t++) {
                answer[i][j] = answer[i][j] + arr1[i][t] * arr2[t][j]
            }
        }
    }
    return answer;
}